cli/create-scss.js

```diff
@@ -34,6 +34,7 @@ var writeScss = function(iconInfos, scssPath) {
 		var iconInfo = iconInfos[i];
 
 		scss.write('@mixin ' + iconInfo.mixin + '() {\n');
+		scss.write('	@include d2l-icon();\n');
```

Comment:
That's sort of what I was thinking too - and as apps switch away from the Sass this duplication will be limited and eventually gone... ?